common/libs/wallet/deterministic.ts

```diff
@@ -1,7 +1,7 @@
 import { IWallet } from './IWallet';
 
 export default class DeterministicWallet {
-  private address: string;
+  public address: string;
```

Comment:
Address should remain private / protected, since we have getAddress.

Reply:
Whoops, my bad. Will revert to `private` and use `getAddress()` instead.